input/pagecontent/changes.md

```diff
@@ -8,3 +8,7 @@ This change log documents the significant updates and resolutions implemented fr
 #### Changes in this version
 - New profiles: 
   - AU eRequesting Task ([FHIR-46947](https://jira.hl7.org/browse/FHIR-46947), [FHIR-46950](https://jira.hl7.org/browse/FHIR-46950), [FHIR-46956](https://jira.hl7.org/browse/FHIR-46956), [FHIR-46958](https://jira.hl7.org/browse/FHIR-46958), [FHIR-46959](https://jira.hl7.org/browse/FHIR-46959), [FHIR-47081](https://jira.hl7.org/browse/FHIR-47081), [FHIR-47082](https://jira.hl7.org/browse/FHIR-47082), [FHIR-47083](https://jira.hl7.org/browse/FHIR-47083), [FHIR-47084](https://jira.hl7.org/browse/FHIR-47084), [FHIR-47086](https://jira.hl7.org/browse/FHIR-47086), [FHIR-47102](https://jira.hl7.org/browse/FHIR-47102).
+
+- AU eRequesting ServiceRequest
+  - Made AUeRequestingServiceRequest.encounter mandatory per [FHIR-47008](https://jira.hl7.org/browse/FHIR-47008)
```

Comment:
AUeRequestingServiceRequest.encounter should be ServiceRequest.encounter. 
Please add "(1..1)" after "mandatory" so it reads "mandatory (1..1)"), and remove "per" (sorry to be picky bit we don't use it in change logs in any of the specs).